deliverables/README.md

```diff
@@ -20,49 +20,89 @@ I seek the Rails
+<br>
 
 ---
 
 ### Section 2: HTML and CSS 101
 
 #### meta
-Why do we separate HTML and CSS?  
+**Why do we separate HTML and CSS?**<br>
+Keeping the languages seperate allows for the code to be more manageable, as well as more efficient and simpler to understand/apply changes when necessary.
+<br>
 
 #### HTML
-What are classes and IDs (and how are they different)?  
-What are elements?  
-What are tags?  
-What are attributes?  
-What are forms?  
-What is a div?  
-
-What are selectors?  
-What are properties?  
-What are values?  
-How do CSS styles for a particular element get inherited? ie. how does an element get its "default" styles?  
-What are two CSS attributes you can change to push an element around on the page?  
-What are the three different ways to include CSS in your project or use CSS to style a particular element?  
+**What are classes and IDs (and how are they different)?**<br>
+Both Classes and IDs are used when styling HTML markup. They are different in that classes define the characteristics of an object in HTML. Classes can be linked to various objects, allowing for them <strong>all</strong> to have consistent formatting. IDs are used for specific, individual elements, rather than various ones. <br>
+
+**What are elements?**<br>
+An HTML element is any content that is bound by a start tag and an end tag.<br>
+**What are tags?**<br>
+Tags are commands, basically, that bind an element. Tags are held between "less than" and "greater than" symbols, and mark the beginning and the end of an element. <br>
+**What are attributes?**<br>
+Attributes provide additional information about an element and are contained in the start tag. <br>
+**What are forms?**<br>
+HTML forms are used to collect user input, and define an HTML form. <br>
+**What is a div?**<br>  
+divs are used in HTML and CSS documents. In HTML, they define a division or section in the document. In CSS, divs group block-elements in order to format them properly. <br>
+
+**What are selectors?**<br>
+Selectors are used in CSS and allow the developer to select and manipulate HTML elements. They can "find" HTML elements based on their ID's, Classes, or attributes. <br>
+**What are properties?**<br>
+Properties are special features that can be utilized to change the formatting of a specific element. These include colors, opacity, borders, lists, columns, etc. <br>
+**What are values?**<br>
+Values are assigned to the properties that give them a specific task. Ex: font-size 14px, or font-family Verdana.<br>
+**How do CSS styles for a particular element get inherited? ie. how does an element get its "default" styles?**<br>
+There are a couple of ways for an element to be inherited; the default browsers markup settings, the styles can be specified by the user, or the styles can be linked to the document by the author, typically through an external style sheet, through a determiend style at the beginning of the document, or within a specific element in the body (inline). <br>
```

Comment:
Basically an element inherits its parent's styles for all properties, unless specific properties are matched via a css rule's selector in which case all properties defined in that rule are applied to the element.